datafusion-cli/Cargo.toml

```diff
@@ -27,7 +27,7 @@ repository = "https://github.com/apache/arrow-datafusion"
 rust-version = "1.58"
 
 [dependencies]
-clap = "2.33"
+clap = "3.0"
```

Comment:
Why somewhere we get clap = "3" but otherwhere we get clap = "3.0" ?

Reply:
This is dependabot's behavior, I haven't studied it.

Reply:
probably a bug in dependabot, in cargo, `3` is the same as `3.0` semantically when it comes to versioning pinning